src/command_modules/azure-cli-network/azure/cli/command_modules/network/_help.py

```diff
@@ -1728,6 +1788,10 @@
 helps['network vnet update'] = """
     type: command
     short-summary: Update a virtual network.
+    examples:
+        - name: Update a specific virtual network's DNS server
```

Comment:
Remove `specific` and replace with either generic language (`a`) or name the virtual network explicitly (`MyVNet`).

Reply:
Fixed in my branch